### ISSUE TITLE
Fewer characters per line and smaller width of content

### ISSUE THREAD

**Issue by sayanee** [org-arl/unet-blog#26]
## Current situation

- too many characters per line which do not make it readable 
- the content is too wide
- the text size is too small

![image](https://user-images.githubusercontent.com/413879/46063580-d60df580-c19f-11e8-9151-e7132107c52e.png)

## Suggested situation

- reduce the content width
- increase the font-size
- ensure it's about `~75` characters per line

Checkout articles on [Medium](https://medium.com/)

## Reference

- [4 REASONS I USE LARGE TYPE](http://mikeyanderson.com/optimal_characters_per_line)

**Comment by notthetup** [comment]
Strange. I'm guessing it's based on screen size? I'm seeing this. 

~88 chars

![screenshot 2018-09-28 07 52 54](https://user-images.githubusercontent.com/553140/46190354-8f311480-c2f3-11e8-9499-2410ab502427.jpg)

**Comment by sayanee** [comment]
Yeah, that is because the width of the content is currently relative. We can make it absolute and fixed.

```css
.posts .post-teaser {
    padding: 4% 20%;
}
```

## Full-width browser on same monitor

![image](https://user-images.githubusercontent.com/413879/46199530-dd6d0480-c341-11e8-9995-b82eff057f0a.png)

## Smaller width browser on same monitor

![image](https://user-images.githubusercontent.com/413879/46199636-21f8a000-c342-11e8-9066-b4f17690184b.png)

## Same content width regardless of browser width

Where as in Medium.com, the content width is fixed:

![image](https://user-images.githubusercontent.com/413879/46199722-510f1180-c342-11e8-8f94-f7bc05568e5d.png)
